html/pfappserver/root/static.alt/src/views/Configuration/_config/network.js

```diff
@@ -140,6 +141,22 @@ export const view = (form = {}, meta = {}) => {
               attrs: attributesFromMeta(meta, 'interfaceSNAT')
             }
           ]
+        },
+        {
+          label: i18n.t('Static routes'),
+          text: i18n.t('Add custom static toutes managed by keepalived, one line per static route.'),
```

Comment:
Although its obvious to me that the format should be the same one as in the route command, saying it and giving an example would be good since this is a text field. That will help users understand what they need to input